squad/frontend/user_settings.py

```diff
@@ -55,8 +57,42 @@ def api_token(request):
     return render(request, 'squad/user_settings/api_token.html', context)
 
 
+@login_required
+def subscriptions(request):
+
+    subscriptions = Subscription.objects.filter(user=request.user)
+    groups = Group.objects.all().prefetch_related('projects')
+
+    if request.method == "POST":
+        for project_id in request.POST.getlist("subscriptions[]"):
+            try:
+                Subscription.objects.create(
```

Comment:
Note that calling `Subscription.objects.create` (instead of using a Django form) bypasses the model validation, maybe we don't want that.

Reply:
This is exactly why we do self._validate_email() in the overriden save method in Subscription model so it's not bypassed. Same would happen if you use API or some other means to create/update this specific model. I can add form here of course but I think it's redundant.